### PR TITLE
Update Grafana upstream version and appVersion

### PR DESCRIPTION
**Pull request by QuantumEnigmaa**

### CODE REVIEW

**Comment by QuantumEnigmaa** [comment]
Tested with app from #81 on WC and MC on `ginger`  and seems to work alright : 
- Grafana deployed and using giantswarm/9.2.5 docker image
- Authentication working 
- Dashboards are there

**Comment by hervenicol** [comment]
Chart.lock was not updated.
During your tests, did you check the deployed version was really the new version?

**Comment by QuantumEnigmaa** [comment]
You have a point.
Chart.lock is now updated and I am going to check and test it again :)

**Comment by QuantumEnigmaa** [comment]
Finished with this new testing :
- Grafana deployed and using giantswarm/9.2.5 docker image
- Grafana helm chart using grafana:6.44.7 upstream chart (specified in every resources labels)
- Authentication working
- Dashboards are there
- Datasource is correct 

Still want to note that the dashboard I checked were displaying "no data" and that on WC no datasource was configured.

**Review by hervenicol — APPROVED** [review]
LGTM

File: helm/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: grafana
 version: 2.0.2
-appVersion: 9.1.1
+appVersion: 9.2.5
 kubeVersion: "^1.8.0-0"
```
Comment (QuentinBisson) [inline helm/grafana/Chart.yaml:5]: Did you not forget to update the version field of this chart?

Comment (QuantumEnigmaa) [reply helm/grafana/Chart.yaml:5]: I'm not sure to understand.
In this chart I updated the app version to 9.2.5 and the version to 6.44.7

Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:5]: https://github.com/giantswarm/grafana-app/blob/6f346e62a6d62bce3a9db3e55e72279c2d327499/helm/grafana/Chart.yaml#L3

Comment (QuantumEnigmaa) [reply helm/grafana/Chart.yaml:5]: Oh my bad ! I thought I would change it after merging this branch and creating a minor release

Comment (hervenicol) [reply helm/grafana/Chart.yaml:5]: Same, I thought ABS would do it automagically at release time.

Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:5]: Oh then no :( That's the issue with ABS, we have to manage the version ourselves

Comment (QuantumEnigmaa) [reply helm/grafana/Chart.yaml:5]: Then I will change it to 2.1.0 is that ok for you ?

Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:5]: Awesome :)

Comment (QuantumEnigmaa) [reply helm/grafana/Chart.yaml:5]: done

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**